### PR TITLE
Some updates to merge rules

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #132296

The added people from metamates don't actually make a material
difference right now but I added some for fun.

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

### CODE REVIEW

File: .github/merge_rules.yaml
Comment (albanD) [inline .github/merge_rules.yaml:528]: Already in here 5 lines above

**Review by albanD — APPROVED** [review]
Sounds good!

**Review by malfet — APPROVED** [review]

**Comment by albanD** [comment]
Welcome to the Core Reviewers group @eqy @jansel @jeffdaily @eellison @anijain2305 @bdhirsh @zou3519 @ptrblck 

As the name suggest, your PR accept is now valid for any part of the codebase !
With great power comes great responsability and so a quick reminder here that your accept means the PR can be merged as is. Do make sure that the PR is valid (A feature we do want), is correct (Does what is advertized without correctness issues), has the appropriate testing (it will continue to be correct) and is not likely to break a lot of users without appropriate care being taken before giving it a final approval.

Thanks a lot for your help and care in keeping PyTorch awesome ! :fire:

**Pull request closed without merging**
